sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/UnifiedAddressRequest.kt

```diff
@@ -0,0 +1,22 @@
+package cash.z.ecc.android.sdk.model
+
+data class UnifiedAddressRequest(val flags: Int) {
+  init {
+    require(flags - ALL == 0) {
```

Comment:
I'm pretty sure this requires that the caller only ever set `ALL`. For the check you actually want, this needs to be a `flags AND (NOT ALL)`.